### PR TITLE
feat: initialize tailwind

### PR DESCRIPTION
**Pull request by nikaakin**

### CODE REVIEW

File: tailwind.config.js
Comment (MilosIl) [inline tailwind.config.js:1]: should we put in custome theme, from figma file? like colors and font sizes and font it self?

**Review by MilosIl — COMMENTED** [review]
agree to use tailwind. it should be merged into dev

File: index.html
Comment (MilosIl) [inline index.html:1]: YouGo, is name of our app

**Review by MilosIl — APPROVED** [review]
i agree we should use tailwind

**Review by MilosIl — APPROVED** [review]

**Review by Ben2oid — APPROVED** [review]

**Pull request merged**